docs/compositions.md

```diff
@@ -0,0 +1,144 @@
+# Action compositions
+
+Action compositions make it possible to dynamically build and invoke a series of actions. Action compositions are similar to sequences. However, whereas the components of a sequence must be specified before invoking the sequence, components of a composition can be decided as the composition is running.
+
+## Example
+
+Suppose we define an _increment_ action:
+
+```
+$ cat > increment.js
+function main({ value }) { return { value: value + 1 } }
+^D
```

Comment:
What are this ^D about ? in the file

Reply:
separate command from code, make code in it's own block with the language easy to copy paste out.
Also I think removing the `cat` command is better, I'm not sure if there is a `cat` in windows.

Example:
Suppose we define an _increment_ action: in a source file `increment.js`
```javascript
function main({ value }) { return { value: value + 1 } }
```
Create the action _increment_:
```
wsk action create increment increment.js
```

Reply:
Updated